tools/clang/test/SemaHLSL/hlsl/linalg/outerproductaccumulate-errors.hlsl

```diff
@@ -0,0 +1,53 @@
+// RUN: %dxc -I %hlsl_headers -T lib_6_9 -enable-16bit-types %s -verify
+
+#include <dx/linalg.h>
+
+RWByteAddressBuffer RWBuf;
+
+// test for inputs of different size
+export void Test4(vector<half, 128> Input1, vector<half, 64> Input2) {
+  using namespace dx::linalg;
+
+  RWMatrixRef<DATA_TYPE_FLOAT16, 128, 64, MATRIX_LAYOUT_OUTER_PRODUCT_OPTIMAL, true>
+      matrix = {RWBuf, 0, 0};
+
+  // clang-format off
+  // expected-error@+5{{no matching function for call to 'OuterProductAccumulate'}}
+  // expected-note@dx/linalg.h:157{{candidate template ignored: could not match 0 against 1}}
+  // __builtin_OuterProductAccumulate(Input1, Input2, RWBuf, 0, DATA_TYPE_FLOAT16, MATRIX_LAYOUT_OUTER_PRODUCT_OPTIMAL, 0);
```

Comment:
```suggestion
```

Reply:
I see this repeated in other tests so I'll stop calling them out. Is it deliberate?

Reply:
I'll remove all commented out builtin calls.